src/Controller/NoteController.php

```diff
@@ -69,6 +69,35 @@ public function index(CaseEntity $case, PaginatorInterface $paginator, NoteRepos
         ]);
     }
 
+    /**
+     * @Route("/new", name="notes_new")
```

Comment:
Consider using `note_new` as route name.

Reply:
Renamed route to `note_new`.